examples/cpp/pyperf/PyPerfNativeStackTrace.cc

```diff
@@ -39,6 +53,11 @@ NativeStackTrace::NativeStackTrace(uint32_t pid, const unsigned char *raw_stack,
   unw_accessors_t my_accessors = _UPT_accessors;
   my_accessors.access_mem = NativeStackTrace::access_mem;
   my_accessors.access_reg = NativeStackTrace::access_reg;
+  char sym[256];
+  ProcSyms* procSymbols = nullptr;
+  // reserve memory for platform-defined path limit AND the symbol
+  size_t buf_size = sizeof(sym) + PATH_MAX + 4;
+  unique_ptr<char[]> buf(new char[buf_size]);
```

Comment:
Btw, any reason not to place it on the stack?

Reply:
No, I guess it can safely be moved to stack. I will fix this.